nomic/dataset.py

```diff
@@ -1356,6 +1356,7 @@ def add_data(
         Args:
             data: A pandas DataFrame, list of dictionaries, or pyarrow Table matching the dataset schema.
             embeddings: A numpy array of embeddings: each row corresponds to a row in the table. Use if you already have embeddings for your datapoints.
+            blobs: A list of image paths, bytes, or PIL Images. Use if you want to add images to your dataset.
```

Comment:
I can clarify this but the distinction is that the AtlasDataset built on top of images uses the image embeddings as the "backbone" (images -> embeddings -> xy coordinates) of the map instead of as supplementary metadata. in the latter case, the text is used as the "backbone"

Reply:
Do you have a preference on where we should clarify this? To me, it makes the most sense to do so in the `Image Dataset` docs and maybe having a small comment here

Reply:
I think "Use if you want to add images to your dataset" does not make it clear whether or not the images are the backbone or extra metadata

Reply:
How do I add images as metadata with embeddings created from text?

Reply:
Does the change in https://github.com/nomic-ai/nomic/pull/318/commits/9f7363ba81a358a48a57691a87bd4255ebad4148 make it clearer?

Reply:
> How do I add images as metadata with embeddings created from text?

this is another one of our undocumented features, I think you can just pass the url